### DISCUSSION TITLE
Bug: Window Moves Back to Previous Monitor

### DISCUSSION THREAD

**Discussion by diego-agustin** [rxhanson/Rectangle#1217] [General]
First, absolutely love this app, so thank you for making it available. I've recently started having an issue where a window will snap back to a different monitor inadvertently, details below.

Setup:
- 3 monitors (left: vertical; middle: landscape; right: landscape/Macbook)
- This issue's most easily reproducible with Photoshop.

Steps:
1. Start with Photoshop on any monitor, for the sake of the example let's say left.
2.  Use hotkeys to move Photoshop to the middle monitor, e.g. ctrl + cmd + right twice.
3. Use option + mouse scroll wheel to zoom into Photoshop. 
Result: The Photoshop window snaps back to the left monitor.

Note that if you drag the window to a different monitor vs. using the Rectangle hotkeys the issue isn't reproducible.

**Comment by rxhanson** [comment]
Thanks for reporting. At first glance, this doesn't seem to be a Rectangle issue. Rectangle will only position windows when commanded to do so, and zooming would not trigger a Rectangle command. This should be reflected in the logs (Hold option and select View Logging... in the menu bar menu).

I suspect this might be some sort of glitch with where the window is positioned or the size of it, which happens to be done by Rectangle in your scenario.

If you do see a log showing that Rectangle is placing the window upon zoom, then let me know and we can dig deeper.
